allennlp/commands/find_learning_rate.py

```diff
@@ -2,24 +2,19 @@
 The ``find-lr`` subcommand can be used to find a good learning rate for a model.
 It requires a configuration file and a directory in
 which to write the results.
-
```

Comment:
I thought sphinx required these blank lines...  I'd be happy to be wrong about that, though.

Reply:
I don't know how these got removed